app/templates/ratemypdf_stats.html

```diff
@@ -0,0 +1,262 @@
+{% macro percent_widget(label, title, description, val, mean, std_dev) -%}
+<div class="mb-3">
+  <h4>{{ title }}
+    {% if description | length > 0 -%}
+    <a href="#{{label}}-info" data-bs-toggle="collapse" role="button" aria-expanded="false" aria-controls="{{label}}-info"><i class="fas fa-plus"></i></a>
```

Comment:
nit: I'd change the plus to the downward triangle, like we have in the docassemble interviews.